virttest/env_process.py

```diff
@@ -498,6 +500,8 @@ def postprocess_image(test, params, image_name, vm_process_status=None):
     restored, removed = (False, False)
     clone_master = params.get("clone_master", None)
     base_dir = params.get("images_base_dir", data_dir.get_data_dir())
+    if params.get("storage_type") == "vhost-vdpa":
+        return
```

Comment:
Why do we add this action for the `vhost-vdpa`? we should consider adding the corresponding action in the `preprocess_image` to keep them symmetry if we really need to handle it in the `postprocess_image`

Reply:
I think that the `preprocess_image` and `postprocess_image` handle all the storage types, so we should keep the same workflow for them. 
So I suggest that we can refer to the `preprocess_image`, and use the parameters of the image to control if skip this part for `vhost-vdpa ` storage type. 
We could initialize the instance of `qemu_storage.QemuImg` in each `if` condition, like `preprocess_image`

Reply:
> I think that the `preprocess_image` and `postprocess_image` handle all the storage types, so we should keep the same workflow for them. So I suggest that we can refer to the `preprocess_image`, and use the parameters of the image to control if skip this part for `vhost-vdpa ` storage type. We could initialize the instance of `qemu_storage.QemuImg` in each `if` condition, like `preprocess_image` CC @zhencliu @luckyh

Hello @YongxueHong ,
After I made a check, `check_image()` was called at `_call_image_func()` at preprocess stage and `_call_check_image_func()` at postprocess stage. In those stages, the param `storage_type` is NOT existed. That's why I put the 'vhost-vdpa' in the `check_image()`. In addition, the `check_image()` should be called by each image. 

cc @zhencliu @luckyh